### DISCUSSION TITLE
Data lineage

### DISCUSSION THREAD

**Discussion by jsbrittain** [kraemer-lab/GRAPEVNE#99] [Ideas]
As a concept, data lineage and data provenance will be important in any analysis, local or distributed, to ensure that appropriate steps in the processing pipeline have been honoured. Ideas or resources relating to this are welcomed in this discussion post.

This links to #31 .

**Comment by Mougk** [comment]
Visually we could indicate permissions of data access especially in cases when pipelines are developed by multiple institutions/users who have varying levels of access due to patient privacy or commercial data.
